### ISSUE TITLE
Missing tags on existing AWS VPC subnets in kops 1.5.3

### ISSUE THREAD

**Issue by igoratencompass** [kubernetes/kops#2388]
When creating k8s cluster with kops-1.5.3 in existing AWS VPC the `Name` and `KubernetesCluster` tags are not being applied to the subnets. Apparently their values should be something like:

```
    KubernetesCluster = "<cluster-name>"
    Name              = "eu-west-1a.<cluster-name>"
```

Additional to this, a care should be taken for the cases where one or both of these tags already exist for the subnet (especially the `Name` one which is pretty common). Maybe more unique tag keys should be considered to avoid any clash with existing ones. I don't think overwriting existing ones is the correct action to take as people might have some VPC operations depending on the existing subnet tag. I apologize if this is already dealt with in the code.

**Comment by kenden** [comment]
This is a possible duplicate of https://github.com/kubernetes/kops/issues/1142

**Comment by onlydole** [comment]
I am also seeing this issue and had to manually tag my subnets to ensure that ELBs were created within AWS.

**Comment by kenden** [comment]
I just created a new cluster with Kops 1.6.1 without any tags on the existing subnets and my ELB registered to all subnets. The ELB has the tags:
`KubernetesCluster: k8s.prod.eu-central-1.redacted.net`
`kubernetes.io/cluster/k8s.prod.eu-central-1.redacted.net: owned`
This new tag `kubernetes.io/cluster/...`, from [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/tags.go#L30) seems to have helped

**Comment by adrianmkng** [comment]
I've just created a K8 cluster using kops 1.6.0 on my subnets which had no tags. No additional tags were applied to my subnets after have deploying the K8 cluster on it. 
Next I created a `LoadBalancer` service type on the K8 cluster I could see it failed to register all the subnets to my ELB.
As I understand it, the existing subnets should have the tags `kubernetes.io/cluster/... : shared` applied to it so that the ELB is aware that the subnets are being managed by kops. Is this understanding correct?

**Comment by kenden** [comment]
@adrianmkng yes that's correct. My [last comment](https://github.com/kubernetes/kops/issues/2388#issuecomment-308029577) was incorrect unfortunately. I edited it.
I think we should close either this ticket or https://github.com/kubernetes/kops/issues/1142.

They are both related to https://github.com/kubernetes/kubernetes/issues/42957 and https://github.com/kubernetes/kubernetes/issues/42963

**Comment by geojaz** [comment]
Closed in #3682
